parlai/mturk/tasks/acute_eval/analysis.py

```diff
@@ -54,6 +54,30 @@ def setup_args():
         default=None,
         help='path to the acute analysis pairs for the corresponding run id',
     )
+    parser.add_argument(
+        '--rounding-digit',
+        type=int,
+        default=2,
+        help='number of digits for rounding displayed table',
+    )
+    parser.add_argument(
+        '--max-matchups-html',
+        type=int,
+        default=10,
+        help='max number of matchups to display per model pair in html',
+    )
+    parser.add_argument(
+        '--min-dialogue-length',
+        type=int,
+        default=-1,
+        help='the minimum number of turns for both dialogues in a matchup to be counted as valid for analysis',
+    )
+    parser.add_argument(
+        '--cherry-pick',
```

Comment:
i know we refer to this as "cherry picking" but it might be better to formalize it a little better, e.g. "annotate-convo" or something (since it's a little unclear what a "cherry-picking" checkbox column is)

Reply:
I c you point. Thanks for the suggestion 👍